styles/library.scss

```diff
@@ -5,6 +5,10 @@
 @layer components {
   // Support some basic text styling in Library components
   .styled-text {
+    :where(a):not(:where([class~='unstyled-text'] *)) {
+      @apply text-brand underline hover:text-brand-dark transition-colors;
+    }
+
```

Comment:
I should break this out into a separate commit. The documentation (pattern library page) for this component used a basic `a` tag and it needed some styling for underline and coloring (i.e. this was the first time I actually used an `a` in the docs and ran into a styling gap).